postgres/src/main/java/org/jdbi/v3/postgres/PostgresPlugin.java

```diff
@@ -136,7 +137,7 @@ public void customizeJdbi(Jdbi jdbi) {
         jdbi.registerColumnMapper(new PGobjectColumnMapperFactory());
 
         // legacy unqualified HSTORE
-        jdbi.registerArgument(new HStoreArgumentFactory()::build);
+        jdbi.registerArgument((ArgumentFactory) new HStoreArgumentFactory()::build);
```

Comment:
I know I'm coming to this weeks after the fact, but why does this require a cast?

Reply:
I wondered too, it didn't make any sense to me at the time since the different possibilities are incompatible with each other